guides/gatherers.md

```diff
@@ -163,48 +220,60 @@ This gatherer allows accessing the hostnames that are resolvable through `/etc/h
 does **not** use domain resolution in any way but instead directly parses the file.
 
 It allows one argument to be specified or none at all:
- - When a hostname is provided as an argument, the gatherer will return an array of IPv4 and/or IPv6 addresses.
- - When no argument is provided, the gatherer will return a map with hostname as keys and arrays with IPv4 and/or IPv6 addresses.
 
-Sample arguments:
+- When a hostname is provided as an argument, the gatherer will return an array of IPv4 and/or IPv6 addresses.
+- When no argument is provided, the gatherer will return a map with hostname as keys and arrays with IPv4 and/or IPv6 addresses.
 
-| Name                                | Return value                                                                                            |
-|:------------------------------------|:--------------------------------------------------------------------------------------------------------|
-|`localhost`                          | list of IPs resolving `localhost` e.g. `["127.0.0.1", "::1"]`                                           |
-|`node1`                              | list of IPs resolving `node1` e.g. `["172.22.0.1"]`                                                     |
-|`no argument provided`               | map with hostnames and IP addresses e.g. `{"localhost": ["127.0.0.1", "::1"], "node1": ["172.22.0.1"]}` |
+Example arguments:
+
+| Name                   | Return value                                                                                            |
+| :--------------------- | :------------------------------------------------------------------------------------------------------ |
+| `localhost`            | list of IPs resolving `localhost` e.g. `["127.0.0.1", "::1"]`                                           |
```

Comment:
Once you fix the "Example output" for the hosts gatherers, I'd remove the `e.g.` part from here (to follow what we have done elsewhere and avoid duplication)